src/redux/reducers/userReducer.js

```diff
@@ -49,6 +53,20 @@ export default createReducer(intialState, {
   [logoutError]: (state, { payload }) => ({
     ...state,
     error:
-      payload?.full_messages?.[0] || 'An error occurred. Please try again.' 
+      payload?.full_messages?.[0] || 'An error occurred. Please try again.'
+  }),
+  [updatePasswordSuccess]: (state, { payload }) => ({ value: payload }),
```

Comment:
You are removing `auth` from the state auth here, is that on purpose?

Reply:
No, it's a mistake.